docker/Dockerfile-netty-centos6

```diff
@@ -0,0 +1,22 @@
+FROM centos:6
+MAINTAINER netty@googlegroups.com
+ENTRYPOINT /bin/bash
+
+ENV SOURCE_DIR $HOME/source
+ENV MAVEN_VERSION 3.5.2
+ENV JAVA_VERSION 1.8.0
+
+RUN mkdir $SOURCE_DIR
+WORKDIR $SOURCE_DIR
+
+# install dependencies
+RUN yum install -y lsb-core autoconf automake libtool make tar glibc-devel openssl-devel apr-devel lksctp-tools java-$JAVA_VERSION-openjdk-devel git wget
+
+RUN wget -q http://archive.apache.org/dist/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz
+RUN tar xfz apache-maven-$MAVEN_VERSION-bin.tar.gz
+RUN mv apache-maven-$MAVEN_VERSION /opt/
```

Comment:
Consider merging this into one `RUN ... && \ ...` command to have a single layer created for this group of operation that is logically related.

Reply:
I don't see Maven used anywhere, it has to be put on the path. Here's how to do that:
```
curl -sSL http://archive.apache.org/dist/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz | tar xz && \
ln -sf $(pwd)/apache-maven-$MAVEN_VERSION $(pwd)/maven && \
ln -sf $(pwd)/maven/bin/mvn /usr/local/bin/mvn && \
find maven/bin -type f -name "*.cmd" -exec rm -f {} +
```

Reply:
It's defined in PATH below.